ltp/data.py

```diff
@@ -90,28 +99,57 @@ def arguments(self):
         """
         return self._args
 
+    @property
+    def parallelizable(self):
+        """
+        If True, test can be run in parallel.
+        """
+        return self._parallelizable
+
 
-def read_runtest(suite_name: str, content: str) -> Suite:
+PARALLEL_BLACKLIST = [
+    "needs_root",
```

Comment:
I think we also need save_restore, because sometimes this is set, but not needs_root.

Reply:
@richiejp we discussed a few meetings ago about avoiding tests which need root privileges.

Reply:
yes, I'm not suggesting we remove needs_root. I am saying we also need save_restore.